dreem/inference/tracker.py

```
@@ -188,19 +192,21 @@ def sliding_inference(
                     )
 
                     curr_track_id = 0
+                    # if track ids exist from another tracking program i.e. sleap, init with those
                     for i, instance in enumerate(frames[batch_idx].instances):
```

Ruff

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)